packages/typescript/internal/ts_project.bzl

```diff
@@ -115,7 +118,7 @@ def _ts_project_impl(ctx):
     if ctx.outputs.buildinfo_out:
         outputs.append(ctx.outputs.buildinfo_out)
     runtime_outputs = depset(json_outs + ctx.outputs.js_outs + ctx.outputs.map_outs)
-    typings_outputs = ctx.outputs.typings_outs + [s for s in ctx.files.srcs if s.path.endswith(".d.ts")]
+    typings_outputs = ctx.outputs.typings_outs + ctx.outputs.typing_maps_outs + [s for s in ctx.files.srcs if s.path.endswith(".d.ts")]
```

Comment:
why make this change? I don't think .map files belong in DeclarationInfo, they are meant to be consumed by editors not by other tsc actions that need to type-check

Reply:
checked with @gregmagolan - you are correct we transport .map files along with the thing they map (it's like that for .js.map files for example)